Wikipedia/Code/PreviewLicenseView.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import UIKit
 import WMF
 
-class PreviewLicenseView: UIView, Themeable {
+class PreviewLicenseView: EmbeddableView, Themeable {
```

Comment:
I thought we talked about replacing this with the approach used on the announcement cell - it would just be a UITextView with a delegate to handle the link that was actually tapped